src/Symfony/Component/Routing/composer.json

```diff
@@ -38,7 +38,7 @@
         "symfony/yaml": "For using the YAML loader",
         "symfony/expression-language": "For using expression matching",
         "doctrine/annotations": "For using the annotation loader",
-        "symfony/dependency-injection": "For loading routes from a service"
+        "symfony/dependency-injection": "For using the routing resolver compiler pass"
```

Comment:
Not sure about this one : do we promote other components, or tell what can be done with them?

Reply:
I would remove it altogether.

Reply:
Didn't get the chance to do it 😅 @javiereguiluz This could be a good "first contribution issue".

Reply:
I've done it :)